facia/test/FaciaControllerTest.scala

```diff
@@ -224,4 +225,23 @@ import scala.concurrent.Await
     responseHeaders("Surrogate-Control") should include("max-age=900")
   }
 
+  // TODO - devise a test that works!
+  it should "render email fronts including the TiF podcast with story-related image and not the default album art" in {
+    conf.switches.Switches.FlagshipEmailContainerSwitch.switchOn()
```

Comment:
Should these switches be restored to their previous state at the end of the test?

Reply:
I've removed this test so no longer a problem :)